addons/binding/org.openhab.binding.hyperion/README.md

```diff
@@ -59,56 +59,62 @@ Thing hyperion:serverNG:myServer [ host="192.168.0.10", port=19444, priority=50,
+| hyperionenabled | Switch | Enables or disables Hyperion                       | hyperion:serverNG:myServer:hyperionenabled |
+| blackborder     | Switch | Enables or disables the black border component     | hyperion:serverNG:myServer:blackborder     |
+| smoothing       | Switch | Enables or disables the smoothing component        | hyperion:serverNG:myServer:smoothing       |
+| kodichecker     | Switch | Enables or disables the kodi checker component     | hyperion:serverNG:myServer:kodichecker     |
+| forwarder       | Switch | Enables or disables the forwarder component        | hyperion:serverNG:myServer:forwarder       |
+| udplistener     | Switch | Enables or disables the udp listener component     | hyperion:serverNG:myServer:udplistener     |
+| boblightserver  | Switch | Enables or disables the boblight server component  | hyperion:serverNG:myServer:boblightserver  |
+| grabber         | Switch | Enables or disables the grabber component          | hyperion:serverNG:myServer:grabber         |
+| v4l             | Switch | Enables or disables the V4L component              | hyperion:serverNG:myServer:v4l             |
+| leddevice       | Switch | Enables or disables the led device component       | hyperion:serverNG:myServer:leddevice       |
 
 ## Items:
 
+### Hyperion Server (V1):
+
 ```
-Dimmer Brightness 			"Brightness [%s]" 	{ channel="hyperion:serverV1:myServer:brightness"}
-String Effect				"Current effect [%s]" 	{ channel="hyperion:serverV1:myServer:effect"}
-Color MyColor				"Color"	 { channel="hyperion:serverV1:myServer:color"}
-String Clear          "Clear priority"  { channel="hyperion:serverV1:myServer:clear"}
+Dimmer	Brightness	"Brightness [%s]"	{channel="hyperion:serverV1:myServer:brightness"}
+Color	MyColor		"Color"			{channel="hyperion:serverV1:myServer:color"}
+String	Effect		"Current effect [%s]"	{channel="hyperion:serverV1:myServer:effect"}
+String	Clear		"Clear priority"	{channel="hyperion:serverV1:myServer:clear"}
 ```
 
+### Hyperion.ng Server
+
 ```
-Switch HyperionEnabled {channel="hyperion:serverNG:myServer:hyperionenabled"}
-Switch BlackBorderEnabled {channel="hyperion:serverNG:myServer:blackborder"}
-Switch SmoothingEnabled {channel="hyperion:serverNG:myServer:smoothing"}
-Switch KodiCheckerEnabled {channel="hyperion:serverNG:myServer:kodichecker"}
-Switch ForwarderEnabled {channel="hyperion:serverNG:myServer:forwarder"}
-Switch UdpListenerEnabled {channel="hyperion:serverNG:myServer:udplistener"}
-Switch BoblightEnabled {channel="hyperion:serverNG:myServer:boblightserver"}
-Switch GrabberEnabled {channel="hyperion:serverNG:myServer:grabber"}
-Switch V4lEnabled {channel="hyperion:serverNG:myServer:v4l"}
-Switch LedDeviceEnabled {channel="hyperion:serverNG:myServer:leddevice"}
+Dimmer	Brightness		"Brightness [%s]"	{channel="hyperion:serverNG:myServer:brightness"}
+Color	MyColor			"Color"			{channel="hyperion:serverNG:myServer:color"}
```

Comment:
this formatting is not inline. Although it looks good when having clean columns I wouldn't mind to have them all just `space` separated.